### ISSUE TITLE
Hangs on "Ensure k3s initial control plane server is started" if k3s was previously installed but is not running

### ISSUE THREAD

**Issue by gloomytrousers** [PyratLabs/ansible-role-k3s#152]
### Summary

I've got an existing install (via this role) of k3s, but I had manually stopped it on all 3 instances on my cluster. When I run the playbook, it hangs at "Ensure k3s initial control plane server is started" - because the primary won't start if at least one of the secondaries isn't runnning.

Manually starting k3s one or both of the secondaries allows it to continue.

### Issue Type

- Bug Report

### Controller Environment and Configuration

```text
# Begin ANSIBLE VERSION
ansible 2.9.25
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/russ/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.9/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.9.7 (default, Aug 30 2021, 00:00:00) [GCC 10.3.1 20210422 (Red Hat 10.3.1-1)]
# End ANSIBLE VERSION

# Begin ANSIBLE CONFIG

# End ANSIBLE CONFIG

# Begin ANSIBLE ROLES
# /home/russ/.ansible/roles
- xanmanning.k3s, v2.11.0
# /usr/share/ansible/roles
# /etc/ansible/roles
# End ANSIBLE ROLES

# Begin PLAY HOSTS
["pi1.example.com", "pi2.example.com", "pi3.example.com"]
# End PLAY HOSTS

# Begin K3S ROLE CONFIG
## pi1.example.com
k3s_control_node: true
k3s_etcd_datastore: true
k3s_release_version: "stable"
k3s_server: {"cluster-domain": "k3scluster.example.com", "tls-san": "k3scontrol.example.com", "etcd-snapshot-dir": "/mnt/vol1/backups/k3s-etcd-snapshots/pi1", "disable": ["local-storage", "metrics-server"]}
k3s_check_cgroup_option: {"changed": false, "end": "2021-09-19 16:05:26.165217", "stdout": "cpuset\t2\t8\t1", "cmd": ["grep", "-E", "^cpuset\\s+.*\\s+1$", "/proc/cgroups"], "rc": 0, "start": "2021-09-19 16:05:26.153676", "stderr": "", "delta": "0:00:00.011541", "stdout_lines": ["cpuset\t2\t8\t1"], "stderr_lines": [], "failed": false, "failed_when_result": false}

## pi2.example.com
k3s_control_node: true
k3s_etcd_datastore: true
k3s_release_version: "stable"
k3s_server: {"cluster-domain": "k3scluster.example.com", "tls-san": "k3scontrol.example.com", "etcd-snapshot-dir": "/mnt/vol1/backups/k3s-etcd-snapshots/pi2", "disable": ["local-storage", "metrics-server"]}
k3s_check_cgroup_option: {"changed": false, "end": "2021-09-19 16:05:26.191604", "stdout": "cpuset\t2\t14\t1", "cmd": ["grep", "-E", "^cpuset\\s+.*\\s+1$", "/proc/cgroups"], "rc": 0, "start": "2021-09-19 16:05:26.180462", "stderr": "", "delta": "0:00:00.011142", "stdout_lines": ["cpuset\t2\t14\t1"], "stderr_lines": [], "failed": false, "failed_when_result": false}

## pi3.example.com
k3s_control_node: true
k3s_etcd_datastore: true
k3s_release_version: "stable"
k3s_server: {"cluster-domain": "k3scluster.example.com", "tls-san": "k3scontrol.example.com", "etcd-snapshot-dir": "/mnt/vol1/backups/k3s-etcd-snapshots/pi3", "disable": ["local-storage", "metrics-server"]}
k3s_check_cgroup_option: {"changed": false, "end": "2021-09-19 16:05:26.202459", "stdout": "cpuset\t6\t8\t1", "cmd": ["grep", "-E", "^cpuset\\s+.*\\s+1$", "/proc/cgroups"], "rc": 0, "start": "2021-09-19 16:05:26.191115", "stderr": "", "delta": "0:00:00.011344", "stdout_lines": ["cpuset\t6\t8\t1"], "stderr_lines": [], "failed": false, "failed_when_result": false}

# End K3S ROLE CONFIG

# Begin K3S RUNTIME CONFIG
## pi1.example.com
## pi2.example.com
## pi3.example.com
# End K3S RUNTIME CONFIG
```

### Steps to Reproduce

* Run a playbook using the xanmanning.k3s role to install k3s, with
```yaml
k3s_control_node: true
k3s_etcd_datastore: true
```
* Manually stop k3s on all nodes
* Run the playbook again

### Expected Result

Playbook should run to the end, starting secondaries if necessary.

### Actual Result

```text
TASK [xanmanning.k3s : Ensure k3s initial control plane server is started] ********************************************************************
skipping: [pi2.example.com] => {"changed": false, "skip_reason": "Conditional result was False"}
skipping: [pi3.example.com] => {"changed": false, "skip_reason": "Conditional result was False"}
[***hangs here, until one of secondaries is started manually***]
changed: [pi1.example.com] => {"changed": true, "enabled": true, "failed_when_result": false, "name": "k3s", "state": "started" [...snip]}
```

**Comment by xanmanning** [comment]
Sooooo... You need to start a minimum of two nodes before anything else can start?

Once provisioned the "primary" label is kind of pointless in a 3 node Etcd, the first to be created will very unlikely remain the leader over time.

**Comment by xanmanning** [comment]
I will look at this when I get time

**Comment by xanmanning** [comment]
Should be fixed in `v2.12.0`

**Comment by gloomytrousers** [comment]
Yup, this is fixed, thanks :-)
